### PR TITLE
Need to change unit ranks for group control to be maintained.

### PR DESCRIPTION
**Pull request by costno**

### CODE REVIEW

File: f/briefing/f_orbatNotes.sqf [hunk 1]
```diff
@@ -160,7 +160,10 @@ player createDiaryRecord ["fa3_actions", ["Group Leadership",
 	OVERRIDE GROUP CONTROL: <br/> 
 	 | <execute expression=""
 		if (leader group player != player) then {
-			hintsilent 'Taking control of your group';group player selectLeader player;
+			hintsilent 'Taking control of your group';
+			leader (group player) setUnitRank 'PRIVATE'; 
+			player setUnitRank 'COLONEL';
```
Comment (SamLex) [inline f/briefing/f_orbatNotes.sqf]: Does this change the order of the units e.g. in DUI? Or is that set by the editor / join order?

**Review by SamLex — DISMISSED** [review]

File: f/briefing/f_orbatNotes.sqf (hunk 1 above)
Comment (NikkoJT) [reply f/briefing/f_orbatNotes.sqf]: DUI uses join/creation order for its display order.

**Comment by NikkoJT** [comment]
I know it looks suboptimal for the overall indentation, but you cannot put whitespace in strings, it _does_ show up in the final product.

**Review by SamLex — DISMISSED** [review]

**Comment by NikkoJT** [comment]
Sorry for requesting review at you _again_ Lexer, it needed a conflict resolve

**Review by SamLex — APPROVED** [review]

**Pull request merged**